pom.xml

```diff
@@ -564,6 +564,16 @@
                                         <wait>
                                             <time>40000</time>
                                             <log>Service Broker manager has started</log>
```

Comment:
service broker? maybe this was copied by accident

Reply:
It's intended, this is one of the last lines that appears in the log when SQL server starts:
`2019-11-22 11:40:46.02 spid21s     Service Broker manager has started.`
We assume that SQL server is started when we get that...
They call it "broker"? :man_shrugging: